### PR TITLE
Ajouter l'ADPA 93

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to sgmap/mes-aides-ui#147

### CODE REVIEW

**Comment by MattiSG** [comment]
Can we get rid of the dependency on sgmap/ludwig-api#30?

**Comment by fpagnoux** [comment]
Done

File: lib/config/ludwig.js [hunk 1]
```diff
@@ -144,5 +144,18 @@ module.exports = [
         'imgSrc':'logo_mairie_paris.png',
         'link':'http://www.paris.fr/pratique/toutes-les-aides-et-allocations/aides-sociales/paris-logement-familles-prestation-ville-de-paris/rub_9737_stand_88805_port_24193',
         'isBaseRessourcesYearMoins2':false
+    },
+    {
+        'id':'adpa',
+        'isMontantAnnuel':false,
+        'unite': '%',
```
Comment (MattiSG) [inline lib/config/ludwig.js]: Should be updated based on sgmap/ludwig-ui#51.

Comment (fpagnoux) [inline lib/config/ludwig.js]: Done

**Comment by MattiSG** [comment]
Needs an update of the `ludwig-ui` version number.

**Comment by fpagnoux** [comment]
`mes-aides-api` doesn't depend on `ludwig-ui` though

**Comment by MattiSG** [comment]
Bump it on `mes-aides-ui` then  :unamused:

**Comment by MattiSG** [comment]
GTM

**Pull request merged**